### ISSUE TITLE
Managed identity for OpenAI search and other extensions

### ISSUE THREAD

**Issue by ehrnst** [Azure/azure-sdk-for-net#38516]
Do we have any examples of how to authenticate against azure search through the extension using managed identity instead of `searchKey`?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 81e0efc8-d491-4425-7652-08dddb8e866d
* Version Independent ID: ed418ebd-a705-e133-d8e1-33e85db8fc60
* Content: [Azure OpenAI client library for .NET - Azure for .NET Developers](https://learn.microsoft.com/en-us/dotnet/api/overview/azure/ai.openai-readme?view=azure-dotnet-preview)
* Content Source: [api/overview/azure/preview/ai.openai-readme.md](https://github.com/Azure/azure-docs-sdk-dotnet/blob/master/api/overview/azure/preview/ai.openai-readme.md)
* Service: **openai**
* GitHub Login: @jpalvarezl
* Microsoft Alias: **josealvar**

**Comment by SaurabhSharma-MSFT** [comment]
@ehrnst I don't think this possible and redirecting this to the services team as a feature request.

**Comment by liyuanqiu** [comment]
Any updates?

**Comment by viswanadula8828** [comment]
Any updates?
